stable/elasticsearch/values.yaml

```diff
@@ -29,5 +29,5 @@ DataCpuLimits: "1"
 DataMemoryLimits: "4Gi"
 DataHeapSize: "1536m"
 DataStorage: "30Gi"
-DataStorageClass: "anything"
-DataStorageClassVersion: "alpha"
+# DataStorageClass: "ssd"
+DataTerminationGracePeriodSeconds: 900
```

Comment:
I completely agree and all are very good points. Would you mind informing the base PR https://github.com/kubernetes/charts/pull/890, if we can get this merged, even if still in incubator status one of us can follow up with a new PR with the final changes needed to make this `stable`.